src/routes/index.js

```diff
@@ -4,16 +4,18 @@ import role from './roles'
 import user from './user'
 import product from './products/product'
 import chatRouter from './chat.route'
+import wishlist  from './wishlist'
 
 const router = express.Router()
 
 router.use('/user', auth)
 router.use('/chat', chatRouter)
-router.use('/', role)
 router.use('/user', user)
 router.use('/product', product)
+router.use('/wishlist', wishlist)
 
 router.use('/user', user)
 router.use('/product', product)
```

Comment:
these two  router are being duplicated, I'm suggesting to remove it otherwise looks good